pages/instructorIssues/instructorIssues.ejs

```diff
@@ -176,11 +184,14 @@
               <p class="mb-0">
                 <%= getFormattedMessage(row) %>
               </p>
-              <% if (row.manually_reported) {
-                 var body = encodeURIComponent("Hello " + row.user_name + ",\n\nRegarding the issue of:\n\n\"" + (row.student_message || '-') +"\"\n\nWe've..."); %>
-              <small class="text-muted mr-2">#<%= row.issue_id %> reported <span title="<%= row.formatted_date %>"><%= row.relative_date %></span> by <%= row.user_name || '-' %> (<a href="mailto:<%= row.user_uid || '-' %>?subject=Reported%20PrairieLearn%20Issue&body=<%= body %>"><%= row.user_uid || '-' %></a>)</small>
-              <% } else { %>
-              <small class="text-muted mr-2">#<%= row.issue_id %> reported <span title="<%= row.formatted_date %>"><%= row.relative_date %></span> for <%= row.user_name || '-' %> (<a href="mailto:<%= row.user_uid || '-' %>?subject=Reported%20PrairieLearn%20Issue&body=<%= body %>"><%= row.user_uid || '-' %></a>)</small>
+              <small class="text-muted mr-2">
+                #<%= row.issue_id %> reported <span title="<%= row.formatted_date %>"><%= row.relative_date %></span>
+                <% if (row.show_user) { %>
+                  <% if (row.manually_reported) { %>by<% } else { %>for<% } %>
+                  <% let body = encodeURIComponent("Hello " + row.user_name + ",\n\nRegarding the issue of:\n\n\"" + (row.student_message || '-') +"\"\n\nWe've..."); %>
+                  <%= row.user_name || '-' %> (<a href="mailto:<%= row.user_uid || '-' %>?subject=Reported%20PrairieLearn%20Issue&body=<%= body %>"><%= row.user_uid || '-' %></a>)
```

Comment:
To resolve a conflict as part of the merge 0e0228508d961a74056cb24a13cefeb57788cec4, I added the `let body = ` line and used it as `<%= body %>` in the line above. This change was introduced in PR #3314 (commit 61b7214fe3b22eb4f24ae2164ab0e8b7feee8a01).